clients/filter/filter.go

```diff
@@ -82,7 +86,7 @@ func (c *Client) Healthcheck() (string, error) {
 }
 
 // GetOutput returns a filter output job for a given filter output id
-func (c *Client) GetOutput(ctx context.Context, filterOutputID string) (m Model, err error) {
+func (c *Client) GetOutput(ctx context.Context, filterOutputID string, serviceAuthToken string, downloadServiceToken string) (m Model, err error) {
```

Comment:
Consistency comment: I think the param order for all client methods should be 
`ctx, serviceAuthToken, ...` followed by whatever else is needed.